src/Pos/Generator/Block/Mode.hs

```diff
@@ -317,7 +343,11 @@ instance MonadBlockGenBase m => MonadSlots (BlockGenMode m) where
     currentTimeSlotting = currentTimeSlottingSimple
 
 instance MonadBlockGenBase m => DB.MonadGState (BlockGenMode m) where
-    gsAdoptedBVData = gsAdoptedBVDataDefault
+    gsAdoptedBVData =
+        -- Currently we can't and don't want to test the system in boot mode
+        -- since in requires transaction to have specific distribution.
+        gsAdoptedBVDataDefault <&>
+        (\BlockVersionData{..} -> BlockVersionData { bvdUnlockStakeEpoch = 0, ..})
```

Comment:
> Alright, i thought this PR should be merged ASAP but it can be WIP for one more day if you want to. I'll implement boot era payload corrections.

Yes, go ahead. It can wait now, because I am going to work on performance issues, Vlad is going to implement events executor probably, etc. It seems nobody is blocked.
Also I already allowed 4 things which make me sad and I don't want to allow more until at least one of those 4 is resolved, namely:
1. No bootstrap era in dev mode.
2. Bootstrap era logic in tx construction is in web server.
3. Fee logic in tx construction is in web server.
4. Bootstrap era checks are not in Toil.